server/aws-lsp-codewhisperer/src/language-server/securityScan/securityScanDiagnosticsProvider.ts

```diff
@@ -112,22 +114,16 @@ class SecurityScanDiagnosticsProvider {
         return true
     }
 
+    /**
+     * A default hover element for diagnostics is surfaced in the IDE
+     * that displays as `${diagnostic.Code}: ${diagnostic.Description}`
+     *
+     * Creating a custom lsp hover will add content to the IDE's existing hover,
+     * but will not replace the above default display text.
+     */
     handleHover = () => {
```

Comment:
Based on the comment, it seems like we do not need to handle this event then and want to let client internally handle this. Also is this behavior consistent with other clients i.e if more clients are integrated with this server?
Can this handler be fully deleted?

Reply:
we could remove this - however I'd prefer to keep it. future work will likely involve customization here and I'd like to keep this note to avoid the case development for different clients could break one another

Reply:
I will defer to DEXP team and @zixlin7 / @mj632 regarding any concerns about behaviors across diff clients.

Reply:
I think keeping no-op code for potential future use is typically unnecessary, unless we know for sure it will be implemented. When there is a use case for hover handling, it can be implemented.

Reply:
okay - removed